src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -1195,6 +1253,43 @@ input ParkingFilter {
     select: [ParkingFilterOperation!]
 }
 
+input PlanDebugSettings {
+    debugItineraryFilter: Boolean
+}
+
+input PlanLocation @oneOf {
+    coordinate: Coordinate
+    """TODO should place string be a scalar?"""
+    place: String
```

Comment:
I would replace place an explicit list of things we support like `stopLocationId : ID` and so on. I am a bit unsure here, because I do not know if there is a UI that reads the user input "as is" and passes it to OTP - but I hope not.

Reply:
Yes I wasn't sure if we should include this either in the current form. I'll do some refactoring so we can have coordinate or some entity ID, and additionally a label for that place.

Reply:
I have done something in https://github.com/opentripplanner/OpenTripPlanner/pull/5185/commits/27e7fb10fa928f560948fed91506eb191d713108 . However, with this model we can't use the oneOf directive without adding another wrapper for the coordinate/stop(/trip) as label can be included with any of those types.